lib/utils/stream.js

```diff
@@ -113,4 +116,8 @@ Stream.prototype.flush = function() {
   this.trigger('done');
 };
 
+Stream.prototype.error = function(error) {
```

Comment:
I think nodejs streams can trigger an error of sorts as well https://nodejs.org/api/stream.html#stream_event_error